src/superqt/collapsible/_collapsible.py

```diff
@@ -63,6 +72,48 @@ def content(self) -> QWidget:
         """Return the current content widget."""
         return self._content
 
+    def set_expanded_icon(self, icon=None):
+        """Set the icon on the toggle button when the widget is expanded."""
+
+        if icon:
+            self._EXPANDED = icon
+        else:
+            pixmap = QPixmap(16, 16)
+            pixmap.fill(Qt.transparent)
+            painter = QPainter(pixmap)
+            color = self._toggle_btn.palette().color(QPalette.WindowText)
+            painter.setPen(color)
+            symbol = "▼"
+            painter.drawText(QRect(0, 0, 16, 16), Qt.AlignCenter, symbol)
+            painter.end()
```

Comment:
All this code is duplicated so we could use a helper private function which just receives the symbol. 

```python
def _convert_symbol_to_icon(self, symbol: str) -> QIcon:
    pixmap = QPixmap(16, 16)
    pixmap.fill(Qt.transparent)
    painter = QPainter(pixmap)
    color = self._toggle_btn.palette().color(QPalette.WindowText)
    painter.setPen(color)
    painter.drawText(QRect(0, 0, 16, 16), Qt.AlignCenter, symbol)
    painter.end()
    return QIcon(pixmap)
```

---

@tlambert03 do we want to support symbols as text in general or limiting the approach to icons only is ok?

Reply:
Also not sure how the icon size, will play out with custom QSS 🤔

Reply:
I just know that this doesn't seem to work unless I have a size in there.  16 seemed to look the best.  How would you suggest I handle this?

Reply:
Maybe get the size of the QPushButton or the default font size and use that

Reply:
ok. will look at doing that.